doc/backend_vars.asciidoc

```diff
@@ -272,6 +273,11 @@ GENERAL_HW_POWEROFF_CMD;string;;Shell Command to power off the SUT (in CMD_DIR)
 GENERAL_HW_POWEROFF_ARGS;string;;Arguments to pass GENERAL_HW_POWEROFF_CMD Shell script
 GENERAL_HW_FLASH_CMD;string;;Shell Command to flash a disk image on SUT (in CMD_DIR), optional
 GENERAL_HW_FLASH_ARGS;string;;Arguments to pass GENERAL_HW_FLASH_CMD Shell script
+GENERAL_HW_STORE_ASSET_CMD;string;;Shell Command to extract disk image from SUT (in CMD_DIR), optional
+GENERAL_HW_STORE_ASSET_ARGS;string;;Arguments to pass GENERAL_HW_STORE_ASSET_CMD Shell script. The script will get also extra arguments: disk number and file path to save it into.
```

Comment:
Pun intended? 😉 The descriptions already refer to the disk image, so maybe `IMAGE` is a more descriptive term to use here and a single word fits in better with existing ones?
```suggestion
GENERAL_HW_IMAGE_ARGS;string;;Arguments to pass GENERAL_HW_IMAGE_CMD Shell script. The script will get also extra arguments: disk number and file path to save it into.
```